--- dpdk/config/common_base	2019-07-08 11:57:28.000000000 +0200
+++ common_base	2019-07-10 15:03:57.000000000 +0200
@@ -38,7 +38,7 @@
 #
 # Compile to share library
 #
-CONFIG_RTE_BUILD_SHARED_LIB=n
+CONFIG_RTE_BUILD_SHARED_LIB=y
 
 #
 # Use newest code breaking previous ABI
@@ -957,3 +957,9 @@
 # Compile the eventdev application
 #
 CONFIG_RTE_APP_EVENTDEV=y
+
+#
+#Complie Xilinx QDMA PMD driver
+#
+CONFIG_RTE_LIBRTE_QDMA_PMD=y
+CONFIG_RTE_LIBRTE_QDMA_DEBUG_DRIVER=y
